src/ui-components/Button/Button.tsx

```diff
@@ -4,7 +4,7 @@ import clsxMerge from '@lib/clsxm'
 
 const ButtonVariant = ['contained', 'outlined', 'text'] as const
 const ButtonColor = ['primary', 'secondary', 'gray'] as const
-const ButtonSize = ['xs', 'sm', 'bs', 'lg', 'fl', 'hg', 'app'] as const
+const ButtonSize = ['xs', 'sm', 'sm2', 'bs', 'lg', 'fl', 'hg', 'app'] as const
```

Comment:
Can we add different naming for sm2 size?

Reply:
done